src/pages/HomePage/index.tsx

```diff
@@ -1,4 +1,17 @@
+import Button from "../../components/atom/button";
+
 const HomePage = () => {
-    return <h1>Home</h1>
-}
+  return (
+    <h1>
+      <Button
+        type="button"
+        text="{Enter button text}"
+        onClick={() => console.log("click")}
+        styles="button-primary-small"
```

Comment:
this is not a style it is className. My suggestion is to have a default style and with additional tw classes, u can change it. So there is no need for the extra lines in the css